### ISSUE TITLE
No audio sound after the gui Launch

### ISSUE THREAD

**Issue by Miniflint** [w-okada/voice-changer#730]
### Issue Type

Bug Report

### vc client version number

MMVCServerSIO_win_onnxgpu-cuda_v.1.5.3.12a.zip

### OS

Windows 10 Pro

### GPU

NVIDIA GeForce RTX 2060

### Clear setting

yes

### Sample model

yes

### Input chunk num

yes

### Wait for a while

The GUI successfully launched.

### read tutorial

yes

### Extract files to a new folder.

yes

### Voice Changer type

RVC

### Model type

one of the base model? (idk what to put there)

### Situation

Hi !
so i've recently started to use the app yet i've noticed that i have no sound on my headphone anymore (speaker still works fine if i put the sound of my headphone)

Basically:
i play a youtube video before opening the app -> i open the app -> when GUI is about to launch i dont have sound anymore on the headphone

Thanks for reading !

### application window capture

i recorded a video if it matters
i can drop it there if needed !

### logs on terminal

it is not a crash so i do not have specific crash logs but here it is:
[vvclient.log](https://github.com/w-okada/voice-changer/files/12374629/vvclient.log)
